package.json

```diff
@@ -15,6 +15,7 @@
     "test": "npm run lerna-test",
     "commit": "commit",
     "package": "npm run package",
+    "setup-local": "npm ci --foreground-scripts && cd examples/cdk && npm ci && cd ../.. && cd examples/sam && npm ci && cd ../..",
```

Comment:
Sorry, I made this out of laziness :-)

Reply:
Maybe we could mention this in this section of the docs? https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/CONTRIBUTING.md?plain=1#L81

Happy to do it in a separate PR if you want, but I think it's great that we have this command. Someone mentioned this in an issue also in the past.